zib/StructureDefinitions/zib-AnatomicalLocation.xml

```diff
@@ -0,0 +1,75 @@
+  <baseDefinition value="http://hl7.org/fhir/StructureDefinition/CodeableConcept" />
+  <derivation value="constraint" />
+  <differential>
+    <element id="CodeableConcept">
+      <path value="CodeableConcept" />
+      <short value="Location" />
+      <definition value="Localisation on/in the body." />
+      <comment value="The zib AnatomicalLocation is mapped to both a profile on BodyStructure (&lt;http://nictiz.nl/fhir/StructureDefinition/zib-AnatomicalLocation-BodyStructure&gt;) and this CodeableConcept profile. The CodeableConcept handles the representation of the location, while the BodyStructure profile covers both location and laterality.&#xD;&#xA;&#xD;&#xA;In cases where only the location is required, or where laterality is embedded within the location code, this CodeableConcept can be used within the resource. However, when laterality needs to be explicitly recorded, the BodyStructure resource should be used, referenced through the BodyStructure Reference within this CodeableConcept. If the BodyStructure reference is used, this CodeableConcept should either remain empty or align with the BodyStructure. element.&#xD;&#xA;&#xD;&#xA;**Note 1:** Receiving systems must support both this CodeableConcept and the BodyStructure resource, as either mechanism may be employed. The rationale for this is that, in many cases, only the location will be required, simplifying the FHIR recording process.&#xD;&#xA;&#xD;&#xA;**Note 2:** Since this is a datatype profile, the binding has been applied to the `.coding` element, as it is not possible to bind the ValueSet to the root element." />
+      <alias value="Locatie" />
+      <mapping>
+        <identity value="zib-anatomicallocation-v1.0.3-2024EN" />
+        <map value="NL-CM:20.7.4" />
+        <comment value="Location" />
+      </mapping>
+      <mapping>
+        <identity value="zib-anatomicallocation-v1.0.3-2024EN" />
+        <map value="NL-CM:20.7.4" />
+        <comment value="AnatomicalLocation" />
+      </mapping>
+    </element>
+    <element id="CodeableConcept.extension:bodyStructureReference">
+      <path value="CodeableConcept.extension" />
+      <sliceName value="bodyStructureReference" />
+      <type>
+        <code value="Extension" />
+        <profile value="http://hl7.org/fhir/StructureDefinition/bodySite" />
+      </type>
+      <isModifier value="false" />
+    </element>
+    <element id="CodeableConcept.extension:bodyStructureReference.value[x]">
+      <path value="CodeableConcept.extension.value[x]" />
+      <type>
+        <code value="Reference" />
+        <targetProfile value="http://nictiz.nl/fhir/StructureDefinition/zib-AnatomicalLocation-BodyStructure" />
+      </type>
+    </element>
+    <element id="CodeableConcept.coding">
+      <path value="CodeableConcept.coding" />
+      <binding>
+        <strength value="required" />
+        <valueSet value="http://decor.nictiz.nl/fhir/ValueSet/2.16.840.1.113883.2.4.3.11.60.121.11.11--20240415000000" />
```

Comment:
I have some second thoughts on binding this ValueSet to `CodeableConcept.coding`. It precludes the use of additional codings, e.g. the following would be impossible.

```xml
<Condition xmlns="http://hl7.org/fhir">
    <meta>
        <profile value="https://example.org/fhir/StructureDefinition/Example-zib-Problem"/>
    </meta>
    <bodySite>
        <coding>
            <system value="http://snomed.info/sct"/>
            <code value="181923006"/>
        </coding>
        <coding>
            <system value="http://loinc.org"/>
            <code value="36023-0"/>
        </coding>
    </bodySite>   
    <subject>
        <reference value="Patient/1234"/>
        <type value="Patient"/>
    </subject>
</Condition>
```

In addition, it would result in a different way to define apply bindings than in the rest of the profiles.

So this makes me wonder: is the problem we're trying to solve really this big that we need to resort to a hackish solution? It's a problem for a select group of people that creates profiles. Couldn't we solve this in another way (perhaps a QA step?)

Reply:
Indeed... this is not desired. We could slice `.coding`... make that slice 1..1 to account for the required binding and the option to add other codes. I think I forgot the slicing here.

I have also asked the question if/why we can't bind anything on the CodeableConcept level: https://chat.fhir.org/#narrow/stream/179166-implementers/topic/ValueSet.20binding.20on.20CodeableConcept.20in.20datatype.20profile 

If this turns out useless, we will add the ValueSet from the host profiles and provide documentation to do so in the profile like we did in 2020. Because I think you are right. It affects a small group of people that have to deal with this.

Reply:
Moved in the direction of setting the binding on the CodeableConcept root.